### DISCUSSION TITLE
Freetube clashes with VPN

### DISCUSSION THREAD

**Discussion by Kan0ba** [FreeTubeApp/FreeTube#6195] [Q&A]
Hi there,

for some time now, FreeTube puts out an API-Error, whenever my VPN is active.  The FreeTube-version I use is up to date and I have no idea, what went wrong, because I have used both VPN and FreeTube before and haven't had an issue. If I turn off the VPN videos play as usual. 
Has anyone an idea how I can solve this problem?
Thank you.

**Comment by absidue** [comment]
YouTube has banned the IP address of the VPN that you are using, you need to try using a different VPN server.

**Comment by Kan0ba** [comment]
Thanks, that worked!
Though it's a bit strange. On Youtube there was no problem with the IP. It only happens, when I use it on FreeTube.
Anyway, as long as it works, that's fine ^-^
